### DISCUSSION TITLE
False Account Flagged

### DISCUSSION THREAD

**Discussion by DragAditya** [community/community#50440] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hey GitHub,
I'm Aditya Few Days Earlier I Contact Your Support Team And They Remove My Flagged Account But in My Dashboard It's Still Showing me That my Account is Flagged,
Please Help Me As Soon As Possible, 

Thanks

![Screenshot_2023-03-19-23-14-50-966_com.android.chrome.jpg](https://github.zendesk.com/attachments/token/9iBO5CF9BGdbMlPpybqOIMtXQ/?name=Screenshot_2023-03-19-23-14-50-966_com.android.chrome.jpg)

**Comment by cehfisher** [comment]
Hi @DragAditya - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @DragAditya ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
